test/FakeStorage/Blob.cs

```diff
@@ -1,145 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using Microsoft.WindowsAzure.Storage;
-using Microsoft.WindowsAzure.Storage.Auth;
-using Microsoft.WindowsAzure.Storage.Blob;
-using Microsoft.WindowsAzure.Storage.Shared.Protocol;
-using Microsoft.WindowsAzure.Storage.Table;
-using System;
-using System.Collections.Generic;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace FakeStorage
-{
-    public class FakeBlobClient : CloudBlobClient
```

Comment:
Where'd this go?

Reply:
It's not used by anything -- we have FakeStorageBlobClient now.